README.md

```diff
@@ -1,13 +1,11 @@
 
-<img width="100%" alt="pipeline" src="https://github.com/user-attachments/assets/11ebfb8d-95d7-43f1-b95d-2f14bf4dac86">
+<img width="100%" alt="pipeline" src="https://github.com/user-attachments/assets/eee02d08-9c76-4a6d-a103-812d7cac81a1">
 <p>
 
-- https://observablehq.com/@jstonge/allotaxonometer-for-all-exp
-- https://allotaxp.vercel.app/
-- :octocat: https://github.com/jstonge/allotaxp
-- :octocat: https://github.com/carterwward/py-allotax
+- Visit our [single-page web application](https://allotax.vercel.app/) to try it out online ([:octocat: github](https://github.com/jstonge/allotax))!
+- Use [py-allotax](https://github.com/carterwward/py-allotax) to use allotaxonometry at scale ([:octocat: github](https://github.com/carterwward/py-allotax))
 
-The primary purpose of the alloxonometer in general is to calculate and visualize the difference between any two Zipfian ranked lists of components. This package provides the utilities to facilitate the creationg of the allotaxonometer in `d3.js`
+`Alloxonometer.js` is a lightweigt Javascript package for combining pairs of Zipfian ranked lists of components and provide [divergence metrics](https://compstorylab.org/allotaxonometry/papers/rank-turbulence-divergence/).
 
```

Comment:
I suggest we add in some of the long foretold documentation (unless you want to do that separately) and close out issues related to updating the docs: "add documentation for users (disambiguate among repos and what they're doing), highlighting tools per use case and extensibility. "

Edit below, just generally:
```suggestion
## Use cases and install
- Explore systems and formulate research questions: we recommend starting with the web app...
- Running multiple or larger scale analyses: we recommend using the py-allotax, which details its install instructions, details the required data format, and provides examples in its repo.
```